modules/jooby-apt/src/test/java/tests/validation/BeanValidationsController.java

```diff
@@ -22,12 +22,6 @@ public Bean validateFormBean(@Valid @FormParam Bean bean) {
     return bean;
   }
 
-  // todo: revive when flash `toNullable` will be fixed
```

Comment:
What I meant here is that it is possible to define the param as object `@FlashParam Bean bean`   
and apt will generate the code that is not executable, since `toNullable` is not there for flash/session/...
<img width="749" alt="image" src="https://github.com/user-attachments/assets/2e6ebd80-292d-4ab4-95f6-754a5811bbcc">

The use-case a bit synthetic, but still, I can add some type checking if you think it makes sense

Reply:
Let's add the type check